vllm/worker/worker_base.py

```diff
@@ -567,6 +567,10 @@ def init_worker(self, all_kwargs: List[Dict[str, Any]]) -> None:
             self.worker = worker_class(**kwargs)
             assert self.worker is not None
 
+    def initialize_from_config(self, kv_cache_configs: List[Any]) -> None:
+        kv_cache_config = kv_cache_configs[self.rpc_rank]
```

Comment:
@ruisearch42 @comaniac FYI, if a method needs to send different argument to different ranks, the indexing should use `self.rpc_rank` , and it should happen in this `WorkerWrapperBase`